### ISSUE TITLE
feat: keep_files should be configurable for `bridgetown deploy` 

### ISSUE THREAD

**Issue by whysthatso** [bridgetownrb/bridgetown#623]
## Summary

add a `keep_files` option to the `deploy` command similar to the existing option in `build`.

## Motivation

the deploy runs   `Bridgetown::Commands::Clean.start` which nukes the destination dir.
i use git to deploy the build artifacts to a destination, and would need the .git dir to stick around.
the `build` command has a `keep_files` option, and i would like to have a similar option for `deploy`

there's been a brief discussion about a similar issue in #232

**Comment by jaredcwhite** [comment]
Yeah, this definitely feels like an oversight/bug. Thanks for filing the issue(s).

**Comment by whysthatso** [comment]
is there a guide how to contribute to the documentation? i could put in a pr possibly.

**Comment by jaredcwhite** [comment]
@whysthatso So the running of the `clean` task is just part of the site's customizable `Rakefile`, so you can definitely just take that out if it's causing an issue. Maybe we can document that option [here](https://github.com/bridgetownrb/bridgetown/blob/main/bridgetown-website/src/_docs/command-line-usage.md#rakefile-and-rake-tasks). I don't think there's a change needed to the internal code.
